src/h5web/visualizations/shared/ButtonGroup.tsx

```diff
@@ -0,0 +1,41 @@
+import React from 'react';
+
+interface Button {
+  label: string;
+  isSelected: boolean;
+  onClick: () => void;
+}
+
+interface Props {
+  buttons: Button[];
+  className?: string;
+  buttonClassName?: string;
+}
+
+function ButtonGroup(props: Props): JSX.Element {
```

Comment:
A tentative implementation of `ButtonGroup`: kept extremely dumb as styles and button functionalities are coming from the props.

Reply:
The toolbar as a whole needs a rethink, so it's perfect for now 💯